6_nodejs/week1/avg.js

```diff
@@ -0,0 +1,29 @@
+import { createRequire } from "module";
+const require = createRequire(import.meta.url);
+
+//process
+
+const args = process.argv;
+
+function average(num1, num2, num3) {
+  const avg = (num1 + num2 + num3) / 3;
+  console.log(avg);
+}
+average(Number(args[2]), Number(args[3]), Number(args[4]));
+
+//minimist
+
+const minimist = require("minimist");
+
+const minimistArgs = minimist(process.argv.slice(2));
+const values = Object.keys(minimistArgs).filter((key) => key !== "_")
```

Comment:
Here again I will suggest to handle potential errors. In that way, you will be able to escape bugs. If the argument is a pure string, fx the argument is "apple" then the code will fail without telling what's wrong. So you can solve that problem by an if statement, where you can check if the argument is not number then throw error with an error message.